### PR TITLE
Advanced options

### PR DESCRIPTION
**Pull request by D-B-Hawk**

### CODE REVIEW

File: containers/clusterForms/clusterCreation/advancedOptions/index.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import React, { FunctionComponent, useState } from 'react';
```
Comment (CristhianF7) [inline containers/clusterForms/clusterCreation/advancedOptions/index.tsx:1]: we already have an advanced options component in the shared folder

File: types/provision/index.ts [hunk 2]
```diff
@@ -21,13 +21,18 @@ export enum ClusterCreationStep {
   DETAILS,
 }
 
-export type NewClusterConfig = {
+export enum ImageRepository {
+  GITHUB = 'github',
+  AWS = 'aws',
+}
```
Comment (CristhianF7) [inline types/provision/index.ts:27]: We only need these values:
```suggestion
export enum ImageRepository {
  GIT = 'git',
  ECR = 'ecr',
}
```

**Review by CristhianF7 — COMMENTED** [review]
Great work dude, I just left some comments

File: containers/clusterForms/clusterCreation/advancedOptions/index.tsx (hunk 1 above)
Comment (D-B-Hawk) [reply containers/clusterForms/clusterCreation/advancedOptions/index.tsx:1]: they are not set up exactly the same unfortuanately

**Review by CristhianF7 — APPROVED** [review]

**Pull request merged**